lib/inflector.php

```diff
@@ -238,7 +238,7 @@ public function camelize($term, $downcase_first_letter = self::UPCASE_FIRST_LETT
 
 				$m = $matches[0];
 
-				return !empty($acronyms[$m]) ? $acronyms[$m] : capitalize($m);
+				return !empty($acronyms[$m]) ? $acronyms[$m] : ucfirst($m);
```

Comment:
I hadn't actually seen that helpers file until now - would that be a more appropriate place to add it?  e.g.

```php
if (!function_exists(__NAMESPACE__ . '\mb_ucfirst'))
{
	function mb_ucfirst($str)
	{
		return upcase(mb_substr($str, 0, 1)) . mb_substr($str, 1);
	}
}
```
Is that naming ok?